monai/engines/evaluator.py

```diff
@@ -85,6 +88,12 @@ def __init__(
             handlers=val_handlers,
             amp=amp,
         )
+        if mode == "eval":
```

Comment:
"eval" and "train" could be put into an Enum called "TrainMode" or something like that which we might add to later.

Reply:
sure, updated in the latest commit.
Thanks.